### ISSUE TITLE
Base Apparel Coming Soon Page - add features per readme

### ISSUE THREAD

**Issue by gracepal** [gracepal/frontend-mentor#36]
Continuation of https://github.com/gracepal/frontend-mentor/issues/33

![coming-soon-page-added-demo](https://github.com/gracepal/frontend-mentor/assets/131278381/f4acd473-d39d-4366-a7bf-cb88e388a7cd)

> from readme:

### The challenge

Users should be able to:

- ⏮️ View the optimal layout for the site depending on their device's screen size 
- ⏮️ See hover states for all interactive elements on the page 
- ✅ Receive an error message when the `form` is submitted if:
  - ✅ The `input` field is empty
  - ✅ The email address is not formatted correctly

> Reference:
⏮️ (in last pr, https://github.com/gracepal/frontend-mentor/pull/34)
✅ (in this pr, TBD)
⏳ (WIP)

### TLDR

- wider box shadow on click (vs. initial set to hover)
- retains input value - does not clear on submit, valid or not
- red checkmark on invalid submission
- green checkmark on valid submission
- checkmarks dynamically +2% left of the submit button 
- clear all green/red indicator once clicked outside form input area
- opacity toggle for error message beneath input in order to retain space on mobile (vs using `display: none` to entirely remove form DOM)
